.evergreen/config.yml

```diff
@@ -792,6 +773,25 @@ functions:
           python oidc_write_orchestration.py
           python oidc_get_tokens.py
 
+  "bootstrap aws-secrets":
+    - command: ec2.assume_role
+      params:
+        role_arn: ${aws_test_secrets_role}
+    - command: shell.exec
+      type: test
+      params:
+        add_expansions_to_env: true
+        working_dir: "src"
+        shell: bash
+        script: |
+          ${PREPARE_SHELL}
+
+          bash ${PROJECT_DIRECTORY}/.evergreen/tox.sh ${PYTHON_BINARY} -m aws-secrets -- "" drivers/test
```

Comment:
I'm assuming this will be changed to something like `atlas_connect`?

Reply:
`run atlas tests` does all of the work now, this function is no longer used.